templates/templates/protecode-guard.yaml

```diff
@@ -42,7 +42,7 @@ presubmits: # runs on PRs
             requests:
               memory: 200Mi
               cpu: 80m
-      run_if_changed: "{{ .run_if_changed }}"
+{{ if .run_if_changed }}      run_if_changed: "{{ .run_if_changed }}"{{ end }}
```

Comment:
I would go against using `if` here. I would rather go with:
```
{{ with .run_if_changed }}run_if_changed: {{ . }}{{ end -}}
```
or at least go with the `{{ end -}}` as this *should* remove the empty line if the condition is not met.

Reply:
I updated this line to use "with" but I think we can't use `{{ end -}}` cause it trims spaces on the next line and `labels` field goes to the left. Anyway I was able to achieve the desired outcome by moving `{{ end }}` to the new line.